tests/files/bugs.parse

```diff
@@ -39,3 +39,13 @@ TS: 0 | 1997-01-01 -99999:-99999:-99999  0Y   0M  -2D /   0H   0M   0S | 1997W01
 TS: 0 | 2004-01-01 05:00:00 GMT 00000  0Y   0M  60D /   0H   0M   0S | 2004W101T05:00+0
 TS: 0 | -99999--99999--99999 -99999:-99999:-99999 | nextyear
 TS: 0 | -99999--99999--99999 -99999:-99999:-99999  1Y   0M   0D /   0H   0M   0S | next year
+TS: 0 | 10000-01-01 00:00:00 | 10000-01-01T00:00:00
```

Comment:
I prefer having these new tests in a new file, say ``tests/files/year-long.parse``. I think it would also be useful to have at least the mega big ones in C-tests (like ``tests/c/issues.cpp``) too.